plasmapy/dispersion/numerical/__init__.py

```diff
@@ -2,4 +2,7 @@
 The `~plasmapy.dispersion.numerical` subpackage contains functionality
 associated with numerical dispersion solvers.
 """
-__all__ = []
+__all__ = [
+  "hollweg",
+]
```

Comment:
```suggestion
__all__ = ["hollweg"]

```

This does not need to be split onto multiple lines until the code reaches the max line limit.  When to start the multiple line split is a little subjective, but it's based on readability and I usually don't split the list until it gets over 3 elements, or it reaches the max line length.